ocs_ci/cleanup/aws/cleanup.py

```diff
@@ -158,20 +169,20 @@ def aws_cleanup():
             "Wrong confirmation answer. Exiting"
         )
 
-    prefixes_hours_to_spare = defaults.CLUSTER_PREFIXES_TO_EXCLUDE_FROM_DELETION
+    prefixes_hours_to_spare = defaults.CLUSTER_PREFIXES_SPECIAL_RULES
 
-    if not args.skip_prefixes:
-        prefixes_hours = input(
-            "Press Enter if there are no cluster prefixes to spare.\n"
-            "If you would like the cleanup to spare specific cluster prefixes, "
-            "please enter them along with the time allowed for these to be kept "
-            "running, in a dictionary representation.\nAn example: "
-            "{\'prefix1\': 36, \'prefix2\': 48}\" "
+    for prefix in args.prefix:
+        _prefix, _hours = prefix.split(':')
+        if not _prefix or not _hours:
+            raise
```

Comment:
Additionally it will be good to check the provided values slightly more better - this check only catch following situation: `--prefix foo:` or `--prefix :10` or `--prefix :`, but lot's of other type of values will cause unprocessed exception:
* `--prefix foo`
* `--prefix foo:bar`
* `--prefix foo:bar:123`

Reply:
Done! Using the type param and a custom validation function was a very good suggestion.